src/main/resources/sql/script_29.03.2024.16_07.sql

```diff
@@ -135,24 +138,24 @@ CREATE TABLE `order`
     `tax`          DECIMAL,
     `shipping_id`  VARCHAR(36),
     `currency`     ENUM ('UAH','EUR','USD')            NOT NULL,
+    `status`       ENUM ('PENDING', 'PROCESSING', 'SHIPPED', 'DELIVERED', 'CANCELED'),
     `created_at`   TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
     FOREIGN KEY (`user_id`) REFERENCES `user` (`id`) ON DELETE CASCADE ON UPDATE CASCADE,
     FOREIGN KEY (`shipping_id`) REFERENCES `shipping_info` (`id`) ON DELETE SET NULL ON UPDATE CASCADE
 );
 
 CREATE TABLE `ordered_item`
 (
-    `id`       VARCHAR(36) PRIMARY KEY,
-    `order_id` VARCHAR(36)  NOT NULL,
-    `item_id`  VARCHAR(36),
-    `title`    VARCHAR(255) NOT NULL,
```

Comment:
Why did you delete the title? If the item ID is deleted, what should we do? We need to have a title for the ordered item at all cases